src/metabase/util/product_feedback.clj

```diff
@@ -0,0 +1,52 @@
+(ns metabase.util.product-feedback
+   (:require
+    [cheshire.core :as json]
+    [clj-http.client :as http]
+    [clojure.string :as str]
+    [environ.core :refer [env]]
+    [metabase.config :as config]
+    [metabase.util :as u]
+    [metabase.util.log :as log]))
+
+(set! *warn-on-reflection* true)
+
+ (def product-feeback-base-url
```

Comment:
You should use `defsetting` for this, then you get the `MB_PRODUCT_...` stuff for free and you can use all of our tooling around mocking settings. Something like

```clj
(defsetting product-feeback-base-url
  "Base url for the product feedback, in DEV it uses a value from an ENV, even if empty, to not send real data to the production endpoint"
  :visibility :internal
  :default ...)
```